common/app/views/fragments/analytics/google.scala.html

```diff
@@ -70,6 +75,7 @@
     @***************************************************************************************
     * Code that is customised by us                                                        *
     ***************************************************************************************@
+    window.guardian.ga.hasInitialised = true;
```

Comment:
I think as this is page level config, it makes sense to live on `window.guardian.config.page.gaIsInitalised` rather than add to the top level guardian object (this isn't very well defined, to be fair!)